src/js/keypress.js

```diff
@@ -9,13 +9,14 @@
 	 * @type {array}
 	 */
     var selectors = [
+		'.list-entries .entry--selected a.entry__title',     // Additional selector for recent Feedly changes
 		'div.selectedEntry a.title',			// title bar for active entry, collapsed or expanded
 		'.selectedEntry a.visitWebsiteButton',	// the button square button on list view
-		'.list-entries .selected a.visitWebsiteButton',	// the button square button on list view
+		'.list-entries .inlineFrame--selected a.visitWebsiteButton',	// (modified @huduchenkov selector) the button square button on list view
```

Comment:
There is no need to have my name in the code :)

```suggestion
		'.list-entries .inlineFrame--selected a.visitWebsiteButton',	// the button square button on list view
```

Reply:
Apologies. I was attempting to give credit, not assign blame. Change accepted.

Reply:
I understand, no worries :)